thundra/integrations/modules/mysql.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import absolute_import
+
+import mysql.connector
```

Comment:
will this fail if mysql dependency is not available because user doesn't use mysql at all?

Reply:
also I didn't see any mysql dependency for development environment